tests/util/models/gpflux/models.py

```diff
@@ -89,6 +90,44 @@ def simple_two_layer_dgp_model(x: TensorType) -> DeepGP:
     return DeepGP([gp_layer_1, gp_layer_2], gpflow.likelihoods.Gaussian(0.01))
 
 
+def separate_independent_kernel_two_layer_dgp_model(x: TensorType) -> DeepGP:
+    if isinstance(x, tf.Tensor):
```

Comment:
you can use https://github.com/secondmind-labs/trieste/blob/develop/trieste/utils/misc.py#L55